addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -259,6 +259,8 @@ def _compute_number_of_days(self):
             if allocation_unit != 'hour':
                 allocation.number_of_days = allocation.number_of_days_display
             else:
+                if not allocation.employee_id._get_hours_per_day(allocation.date_from):
+                    raise ValidationError(_("Set a valid working schedule for employee %(employee)s to enable hour allocation.", employee=allocation.employee_id.name))
```

Comment:
Note for next PRs, I would have kept it simple here, no need of the employee name, you are creating an allocation for an employee (that you selected in the form view), a simple message like `for this employee` would have been enough and no need of `%s`, which always makes translation harder for our translators on transifex, even tho here it's easy to figure.

But don't change it now that it's done